inception/decoder_tpl.go

```diff
@@ -659,9 +664,33 @@ tokerror:
 	}
 	panic("ffjson-generated: unreachable, please report bug.")
 done:
+{{range $index, $field := $si.Fields}}
+	if !ffj_set_{{$si.Name}}_{{$field.Name}} {
+        {{with $fieldName := $field.Name | printf "uj.%s"}}
+	        {{if eq $field.Pointer true}}
+                          {{$fieldName}} = nil
+                {{else if eq $field.Typ.Kind ` + strconv.FormatUint(uint64(reflect.Interface), 10) + `}}
```

Comment:
All of this should be properly indented with tabs so the resulting code doesn't change with gofmt (like the other templates here).